### ISSUE TITLE
superlogcon undocumented

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3154]
Imported from SourceForge on 2024-07-07 14:57:05
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2013-06-20 11:23:00
Original: https://sourceforge.net/p/maxima/bugs/2598

---

The option variable superlogcon isn't documented.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 14:57:07
Created by **[rswarbrick](https://sourceforge.net/u/rswarbrick/)** on 2013-07-01 21:30:59
Original: https://sourceforge.net/p/maxima/bugs/2598/#d30a

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 14:57:10
Created by **[rswarbrick](https://sourceforge.net/u/rswarbrick/)** on 2013-07-01 21:31:00
Original: https://sourceforge.net/p/maxima/bugs/2598/#3d7d

---

Ah. I should probably have read this bug report before deleting the variable in commit [40d26f] that I wrote a couple of days ago.

As far as I can tell, the variables I killed are not used anywhere in the tree and there are no interesting commits in the repository touching a line with the string "superlogcon" in upper or lower case. The only matching commits are one making sysproclaim files, a massive downcasing change, and some disassembly listings from 2003...

Closing since superlogcon no longer exists so *shouldn't* be documented :-)
